restclients_core/dao.py

```diff
@@ -364,11 +365,12 @@ def is_live(self):
 
     def load(self, method, url, headers, body):
         pool = self.get_pool()
-        timeout = pool.timeout.read_timeout
-
         try:
             return pool.urlopen(
-                method, url, body=body, headers=headers, timeout=timeout)
+                method, url, body=body, headers=headers,
+                timeout=pool.timeout, pool_timeout=1)
```

Comment:
Is it necessary to override the timeouts here since they're already applied to the pool?

It doesn't look like pool_timeout will apply since we don't have block set to True: https://urllib3.readthedocs.io/en/latest/reference/urllib3.connectionpool.html#urllib3.HTTPConnectionPool.urlopen  It looks like Timeout.connect will handle the connection timeout.

Reply:
We did set block to True. pool_timeout is option, the default is None.
![image](https://user-images.githubusercontent.com/9834194/122815599-85615380-d28a-11eb-8075-954b76160eb9.png)